src/shared/selectors/flags.ts

```diff
@@ -24,6 +23,7 @@ export const OSS_FLAGS = {
   'load-data-flux-sources': false,
   'load-data-integrations': false,
   'notification-endpoint-telegram': false,
+  editTelegrafs: false,
```

Comment:
we should aim to set these in IDPE and sync it with configcat

Reply:
https://github.com/influxdata/idpe/pull/8992

Reply:
Awesome, these can be removed since they now exist on IDPE